ra_aid/project_info.py

```
@@ -0,0 +1,123 @@
+
+@dataclass
+class ProjectInfo:
+    """Data class containing project information.
+    
+    Attributes:
+        is_new: Whether the project is new/empty
+        files: List of tracked files in the project
+        total_files: Total number of tracked files (before any limit)
+    """
+    is_new: bool
+    files: List[str]
```

Ruff (0.8.2)

Use `list` instead of `List` for type annotation

Replace with `list`

(UP006)